hax/hax/motr/__init__.py

```diff
@@ -297,6 +306,8 @@ def _generate_sub_services(self, note: HaNoteStruct,
         ]
 
     def close(self):
+        LOG.debug('Stopping rconfc')
+        self.stop_rconfc()
```

Comment:
Don't we need to check whether rconfc is currently running?

Reply:
don't think so, `hax/hax/motr/hax.c::stop_rconfc()` handles it,
```
int stop_rconfc(unsigned long long ctx)
{
        struct hax_context *hc = (struct hax_context *)ctx;

        if (hc->hc_rconfc_initialized) {
                struct m0_spiel *spiel = m0_halon_interface_spiel(hc->hc_hi);
                m0_spiel_rconfc_stop(spiel);
        }

        return 0;
}
```